### PR TITLE
Feat: Add more environment variables to datadog

### PR DESCRIPTION
**Pull request by gcallaghan**
#### Which issue this PR fixes
This commit fixes the merge issues in:
  https://github.com/helm/charts/pull/6766

#### Special notes for your reviewer:
I don't have rights to fix the origin PR. This simply fixes the merge issues of the original approved PR as per @davidkarlsen 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/datadog/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: datadog
-version: 1.4.1
+version: 1.4.2
```
Comment (cpanato) [inline stable/datadog/Chart.yaml]: Adding a new functionality. Pleaee bump to 1.5.0

**Comment by gcallaghan** [comment]
@cpanato is this good to go now?

**Comment by gcallaghan** [comment]
@cpanato alright, updated and should be gtg now, hopefully 🤞

**Comment by cpanato** [comment]
@gcallaghan please rebase to fix the conflict

**Pull request closed without merging**